nci/lib/setup_repo.rb

```diff
@@ -59,8 +59,11 @@ def setup_repo!(with_source: false, with_proxy: true, with_install: true)
     # Make sure we have the latest pkg-kde-tools, not whatever is in the image.
     return unless with_install
 
-    raise 'failed to install deps' unless Apt.install(%w[pkg-kde-tools pkg-kde-tools-neon debhelper cmake quilt dh-python dh-translations kde-release-keyring])
-
+    @deps = %w[pkg-kde-tools pkg-kde-tools-neon debhelper cmake quilt dh-python dh-translations])
+      @deps << 'kde-release-keyring' unless JOB_NAME.include?('kde-release-keyring')
```

Comment:
is JOB_NAME actually defined here? I am thinking you may need to pull it out of `env`

Reply:
done